api/MediaList.json

```diff
@@ -20,7 +20,7 @@
             "version_added": true
           },
           "ie": {
-            "version_added": null
+            "version_added": "9"
```

Comment:
I suspect this is later than it was actually introduced. http://mdn-bcd-collector.appspot.com/tests/api/MediaList looks for the interface object, but that's not an important part of the API. A `MediaList` is used on https://developer.mozilla.org/en-US/docs/Web/API/CSSMediaRule which is the object for https://developer.mozilla.org/en-US/docs/Web/CSS/@media.

Can you update these as a group? If `@media` really was supported since IE6, was there no rule object for it at all until IE9, or was it just not exposed?